src/ceph-daemon/ceph-daemon

```diff
@@ -66,6 +66,9 @@ from glob import glob
 
 container_path = None
 
+class CDError(Exception):
```

Comment:
Rename `CDError` -> `Error` since it is already within the scope of the `ceph-daemon` module ...  ?

Reply:
+1